libs/tracing/unix/Tracing.ml

```diff
@@ -51,6 +51,13 @@ module Otel = Opentelemetry
  * TODO we'll probably need instructions for some system of tags?
```

Comment:
Looks like these docs need updating (TODO for endpoint) and possibly instructions on sending to another endpoint with `OTEL_EXPORTER_OTLP_ENDPOINT` if this no longer works after setting the default value in the client config.

Reply:
Good catch, thanks!